modules/distribution/src/repository/conf/datasources/master-datasources.xml

```diff
@@ -46,46 +64,106 @@
             </definition>
         </datasource>
 
+        <!--<datasource>-->
+        <!--<name>WSO2_IDENTITY_DB</name>-->
+        <!--<description>The datasource used for Identity configurations</description>-->
+        <!--<jndiConfig>-->
+        <!--<name>jdbc/WSO2IdentityDB</name>-->
+        <!--</jndiConfig>-->
+        <!--<definition type="RDBMS">-->
+        <!--<configuration>-->
+        <!--<url>jdbc:h2:repository/database/WSO2IDENTITY_DB;DB_CLOSE_ON_EXIT=FALSE</url>-->
+        <!--<username>wso2carbon</username>-->
+        <!--<password>wso2carbon</password>-->
+        <!--<driverClassName>org.h2.Driver</driverClassName>-->
+        <!--<maxActive>50</maxActive>-->
+        <!--<maxWait>60000</maxWait>-->
+        <!--<testOnBorrow>true</testOnBorrow>-->
+        <!--<validationQuery>SELECT 1</validationQuery>-->
+        <!--<validationInterval>30000</validationInterval>-->
+        <!--</configuration>-->
+        <!--</definition>-->
+        <!--</datasource>-->
+
+        <!--<datasource>-->
+        <!--<name>SOCIAL_CACHE</name>-->
+        <!--<description>The datasource used for storing the cached social objects.</description>-->
+        <!--<jndiConfig>-->
+        <!--<name>jdbc/test</name>-->
+        <!--</jndiConfig>-->
+        <!--<definition type="RDBMS">-->
+        <!--<configuration>-->
+        <!--<url>jdbc:h2:repository/database/WSO2SOCIAL_CACHE_DB;DB_CLOSE_ON_EXIT=FALSE</url>-->
+        <!--<username>wso2carbon</username>-->
+        <!--<password>wso2carbon</password>-->
+        <!--<driverClassName>org.h2.Driver</driverClassName>-->
+        <!--<maxActive>50</maxActive>-->
+        <!--<maxWait>60000</maxWait>-->
+        <!--</configuration>-->
+        <!--</definition>-->
+        <!--</datasource>-->
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.